src/includes/class-wordlift.php

```diff
@@ -1308,6 +1334,10 @@ private function define_admin_hooks() {
 		$this->loader->add_action( 'wp_ajax_wl_sample_data_create', $this->sample_data_ajax_adapter, 'create' );
 		$this->loader->add_action( 'wp_ajax_wl_sample_data_delete', $this->sample_data_ajax_adapter, 'delete' );
 
+		// Handle the autocomplete request.
+		add_action( 'wp_ajax_wl_autocomplete', array( $this->autocomplete_adapter, 'wl_autocomplete') );
```

Comment:
Not sure that there will be requests from front-end, so we can remove this if the autocomplete will live in wp-admin